templates/collections/time_period_explorer_page.html

```diff
@@ -54,8 +54,8 @@ <h2 class="featured-article__heading">
         </div>
     </div>
     {% if FEATURE_RELATED_ARTICLE_ON_EXPLORE_PAGES %}
-        {% if page.related_articles %}
-            {% include "includes/related-articles-highlight-cards.html" with articles=page.related_articles title="Records revealed" %}
+        {% if page.related_record_articles %}
+            {% include "includes/related-articles-highlight-cards.html" with articles=page.related_record_articles title="Records revealed" %}
```

Comment:
Cheers Will, could we also add back the original `related_articles` stuff that was originally overwritten?

I think this `related_record_articles` stuff is in the wrong position on the page per the design so will need moving above the `featured_article` on line 9 as well https://www.figma.com/file/uo8YY98lLws8E1HEfpdn5X/TNA-Etna---Design-system?node-id=1145-13318

Suggestion:
```suggestion
{% if page.related_articles %}
    <div class="row">
        <div class="col-md-12">
            {% for article_page in page.related_article %}
                <li>
                    <a href="{% pageurl article_page %}">{{ article_page.title }}</a>
                </li>
            {% endfor %}
        </div>
    </div>
```